docker-compose.development.yml

```diff
@@ -13,6 +13,8 @@ services:
       EMBER_ENABLE_PUBLICATIONS_TAB: "true"
   identifier:
     restart: "no"
+    ports:
```

Comment:
Ik begrijp vanwaar "de vraag" komt, maar dit hoort misschien niet in deze PR thuis. Was dit bedoeld, of is deze mee in een commit geslopen? Ander moeten we samen eens kijken voor het opzetten van een `docker-compose.override.yml` (en een `.env`) file op jouw lokale installatie.

Reply:
Ik heb met @erikap een override file aangemaakt. Bij het bekijken van de files moest ik dit er toen bijzetten en mocht ik dat mee comitten.

Reply:
Prima. Ok :ok: